src/drive/ducks/services/components/SuggestionProvider.jsx

```diff
@@ -53,7 +53,8 @@ class SuggestionProvider extends React.Component {
       const files = allDocs.data
       const folders = files.filter(file => file.type === TYPE_DIRECTORY)
 
-      const notInTrash = file => file.trashed !== true
+      const notInTrash = file =>
+        file.trashed || !/^\/\.cozy_trash/.test(file.path)
```

Comment:
if `file.trashed === true` then `notInTrash` will return `true`. Are you sure?

Reply:
Jesus, no, thank you ><